hil/ext/switches/mock.py

```diff
@@ -56,6 +56,11 @@ def validate(kwargs):
             'password': basestring,
         }).validate(kwargs)
 
+    @staticmethod
+    def validate_port_name(port):
+        """All port names are valid for the mock switch"""
+        pass
```

Comment:
Shouldn't necessarily be the same as the dell switch, but something realistic for a hypothetical switch.